team.html

```diff
@@ -253,6 +254,20 @@ <h5 class="font-weight-bold text-white mb-lg-4 pb-2">Follow Us</h5>
         const tilt = $('.box').tilt({
             scale: 1.01
         });
+
+        const backToTopButton = document.querySelector("#back-to-top-btn");
+        window.addEventListener("scroll", () => {
+            if(window.pageYOffset > 500) { //make the button visible
+                backToTopButton.style.display = "block";
+            }
+            else { // hide the button
+                backToTopButton.style.display = "none";
+            }
+        });
+
+        backToTopButton.addEventListener("click", () => {
+            window.scrollTo(0,0);
+        });
```

Comment:
Extract the common code present here and in `index.html` and keep it in an js file an link that file here, so that there is no repetition of code.
Create a folder named `js` and keep you script there.